server/src/database/migrations/1721115925197-add-jira-auth-option.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner, TableColumn } from "typeorm";
+
+export class Migrations1721115925197 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn(
+      "user_settings",
+      new TableColumn({
+        name: "jiraNotificationIgnore",
+        type: "boolean",
+        isNullable: true,
+      }),
+    );
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.dropColumn("user_settings", "jiraNotificationIgnore");
```

Comment:
Ei tartte muuttaa mitään, mutta ihan FYI, että nää `down`-suunnat voi jättää tyhjiksi. En oo pistäny noita myöskään mun tekemiin migraatioihin.

Funtsin sen niin, että näin pienessä projektissa tuskin tarttee pystyä peruuttamaan migraatioita niin usein, että kannattais käyttää aikaa `down`:n kirjoittamiseen. Ne voi olla joskus vähän monimutkaisia, eikä kaikissa tilanteissa ole edes olemassa idempotenttia käänteisoperaatiota. Käsittääkseni näitä kannattaa kirjoittaa lähinnä silloin, jos joutuu jatkuvasti hyppimään eri branchien välillä, niin saa kannan helposti oikeaan tilaan (jos myös data on sellaista, että se selviää migraatioista molempiin suuntiin).